pom.xml

```diff
@@ -198,6 +198,22 @@ limitations under the License.
         <log4j-silent.properties>file:target/test-classes/log4j-silent.properties</log4j-silent.properties>
 
         <muteTestLogs>false</muteTestLogs>
+        <jdk17JvmArgs>
```

Comment:
a comment with some details on why this is here and when/if all/any of it can be removed in the future would be nice.